http.c

```diff
@@ -1941,6 +1988,25 @@ evhttp_parse_request_line(struct evhttp_request *req, char *line, size_t len)
 		break;
 	} /* switch */
 
+	if ((int)type == EVHTTP_REQ_UNKNOWN_) {
+		/* check extended methods, we only care about the 
+		 * type set by the cmp function if the cmp function
+		 * returns a 0 value.
+		 */
+		struct evhttp_extended_method ext_method;
+
+		ext_method.method = method;
+		ext_method.type = EVHTTP_REQ_UNKNOWN_;
+
+		if (req->evcon->ext_method_cmp &&
+			    req->evcon->ext_method_cmp(&ext_method) == 0) {
+		    /* TODO: make sure the other fields in ext_method are
```

Comment:
I guess we can safely ignore this, since we do not use them later

Reply:
I don't understand what you mean